base/Build.Dockerfile

```diff
@@ -77,6 +77,12 @@ COPY settings ${CONDA_DIR}/share/jupyter/lab/settings
 
 USER root
 
+RUN mkdir /home/ope && \
```

Comment:
i think it could be something like this: 
```
RUN mkdir /home/ope && \
    cd /home/ope && \
    git clone https://github.com/OPEFFORT/tools.git . && \
    ./install.sh && \
    chown jovyan /home/ope && \
    fix-permissions /home/ope
```